src/Chainweb/BlockHeaderDB/RestAPI.hs

```diff
@@ -360,6 +366,7 @@ type BlockHeaderDbApi v c
     :<|> HeaderPutApi v c
     :<|> BranchHashesApi v c
     :<|> BranchHeadersApi v c
+    :<|> PowHeaderApi v c
```

Comment:
If we want to return the pow hash (which I see no value doing), I think, it would make more sense to always include it in the object encoded header. Adding a query parameter for it would be an alternative, but it has the disadvantage that it would apply only to the object encoding.